msa/black-box-tests/src/test/java/org/thingsboard/server/msa/ui/tests/assetProfileSmoke/CreateAssetProfileTest.java

```diff
@@ -167,7 +167,7 @@ public void documentation() {
         String urlPath = "docs/user-guide/asset-profiles/";
 
         sideBarMenuView.openAssetProfiles();
-        profilesPage.allEntity().get(0).click();
+        profilesPage.profileNames().get(0).click();
```

Comment:
assertThat `profilesPage.profileNames()` isNotEmpty() with an alias like " profile names" to have an understandable output instead OutOfRange exception

Reply:
Validation occurs in this custom waiter method:
![image](https://user-images.githubusercontent.com/115694468/224961381-9dad76b2-c211-43d1-92c2-0f25139b57df.png)

Reply:
![image](https://user-images.githubusercontent.com/115694468/224971065-2a997a58-8331-45c0-9cf9-ddd29fef6940.png)